clang/test/CIR/CodeGen/AArch64/neon-misc.c

```diff
@@ -5,11 +5,8 @@
 // RUN:            -emit-llvm -target-feature +neon %s -o %t.ll
 // RUN: FileCheck --check-prefix=LLVM --input-file=%t.ll %s
 
-// This test file contains test cases to those of 
-// clang/test/CodeGen/aarch64-neon-vget.c 
-// The difference is that this file only tests uses vget intrinsics, as we feel
-// it would be proper to have a separate test file testing vset intrinsics 
-// with the file name aarch64-neon-vset.c
+// This test file contains test cases of miscellaneous aarch64 neo intrinsics.
```

Comment:
You should create a separate PR that renames all the aarch64 builtin files, should be a NFC one